### ISSUE TITLE
DS4Windows Not Working with RetroArch

### ISSUE THREAD

**Issue by GoronMegaZord** [libretro/RetroArch#10082]
## Description
Retroarch not receiving key presses from Dualshock 4 (DS4) When Using DS4Windows

### Expected behavior
The controller should activate the keystroke it was bound to in the DS4Windows software.

### Actual behavior

The controller is not receiving keyboard presses from the controller, but will receive presses for the actual game being played.

### Steps to reproduce the bug
1. Get the DS4Windows software and make sure you can play a game using it.
2. Bind Retroarch keyboard hotkey to the controller
3. Witness the behavior not occurring.
4. IMPORTANT: Here is a Youtube video demonstrating the issue: https://youtu.be/tdVlDiib0bU

### Bisect Results
My guess is the program is receiving the key presses from my controller, but is ignoring them. I can't figure out why this is happening.

### Version/Commit
- RetroArch:
Build date: December 5th, 2019. 
Git version: a01e99aa63

### Environment information
- OS: Windows 10 x64 Build 18362 (v10.0)
- Compiler: N/A

**Comment by LordArchantos** [comment]
I'm using DS4Windows as well and don't have any issues with keyboard keys I have mapped to the touchpad (using RA menu and fast forward through it). Do your binds work outside RA (e.g. F1 on the desktop should open Edge with a Bing search for Win10 help)? I did notice your DS4Windows looks different from mine, are you using the original Jays2Kings version? If so, you might want to give Ryochan7's updated branch a try. https://ryochan7.github.io/ds4windows-site/

**Comment by GoronMegaZord** [comment]
Hey there - I appreciate the quick reply! I'm not at my machine right now, but will try the Ryochan7 build when I get home and update with my results. Thanks!

**Comment by GoronMegaZord** [comment]
That did it! Ryochan7 build is working fine with keyboard binds! Thank you, @LordArchantos!

Important note: make sure the "Hide DS4 Controller option is checked" and that the controller is successfully running in exclusive mode. Exclusive mode only worked for me when rebooting my PC, despite already disabling the NVidia overlay. ALSO, it only worked if I connected the controller by pressing the PS button immediately when logging into Windows. Don't let DS4Windows and all your other startup programs load up before pressing the button.

What I actually did was I pressed the PS button to connect right before pressing "Enter" after typing in my password to log in. This way, I ensured the controller was already trying to connect as my startup programs were loading up.

I hope this information helps someone else that either:
A.) Got the wrong build of DS4Windows like myself
B.) Has the right build but is having trouble getting the controller to run in exclusive mode. Exclusive mode is important, because the XBox 360 windows drivers are far more compatible than what the DS4 has.
